### ISSUE TITLE
Can't run unit suite in included config

### ISSUE THREAD

**Issue by LeeShan87** [Codeception/Codeception#5486]
#### What are you trying to achieve?
We have a modular cli application. Some acceptance tests depends on OS available commands. In our CI environment, which is run in docker Ubuntu 16 I only want to run unit and after that functional tests. Acceptance test should run on AWS instances.

I want to achieve when I run:
```
codecept run unit --in-included-too
```
Then only unit/functional/acceptance/other suites should run in an included configuration.

#### What do you get instead?

> Provide console output if related. Use `-vvv` mode for more details.

```bash
# vendor/bin/codecept run unit
Codeception PHP Testing Framework v2.5.6
Powered by PHPUnit 6.5.14 by Sebastian Bergmann and contributors.
Running with seed:


Unit Tests (0) ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------


Time: 3.5 seconds, Memory: 12.00MB


```
> Provide test source code if related

```php
// paste test
```
### Details

* Codeception version: 2.5.6
* PHP Version: 7.2.11-1
* Operating System: Debian 8.8
* Installation type: Composer
* List of installed packages: not related
* Suite configuration:

Main codeception.yml:
```yml
include:
- modules/SomeModule
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
settings:
    colors: true
    strict_xml: true
    memory_limit: 1024M
coverage:
    enabled: true
    include:
        - modules/*/*.php

```
Included module codeception.yml:
```yml
namespace: modules\SomeModule
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
settings:
    bootstrap: _bootstrap.php
    memory_limit: 1024M
    colors: true
    strict_xml: true
coverage:
    enabled: true
    include:
        - src/*.php
    exclude:
        - tests/*
```

PS: I have previously found that this cli option is not implemented. I will try to create a pull request about this issue shortly.

**Comment by vertexvaar** [comment]
Hi there,

i tried to figure this out because i have the exact same use case, so i debugged the code.
I found out that until 4829ad8746ad8f00240395f10a65f1095129d349 the expected behavior was actually default. The change to the code got merged in codeception version 2.1

Just removing the condition does not help, because codeception will execute all suites from any included project so i digged deeper.
I created a PR for 3.1 and 4.0. The new option is called `--recurse-includes`, because of its recursive nature (The method is actually recursive)

I will also add a patch so it can be used with composer-patches.
